<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateMet</class>
 <widget class="QWidget" name="CreateMet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkBoxExtraerMet">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Extract dtm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDeleteDatesMet">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete dates</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonFolderOutMet">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Folder out</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditFolderOutMet">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="labelAmbitoProyectoMet">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Zone project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxAmbitoProyectoMet">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="labelTamanoPixelMet">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Pixel size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTamanoPixelMet">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="labelCoordinateSystemMet">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Coordinate system</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxCoordinateSystemMet">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="checkBoxCortarMet">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Cut DTM</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="labelCortarMet">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Size cut</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTamanyoCortarMet">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QLabel" name="labelNumeroCanalesspasadaMet">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Numbers chanels</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxNumeroCanalesspasadaMet">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="checkBoxFootPrintMaskMet">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>FootPrintMask</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="labelAnchoPasadaMet">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>width flight</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxAnchoPasadaMet">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="labelOffsetPasadaMet">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Offset flight</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxOffsetPasadaMet">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
